### ISSUE TITLE
Properly handle closing window

### ISSUE THREAD

**Issue by Vladimir-csp** [brndnmtthws/conky#312]
When conky window is closed via window manager process prints an error and exits with code 1.

```
XIO:  fatal IO error 11 (Resource temporarily unavailable) on X server ":0"
```

**Comment by plikhari** [comment]
Yes perhaps - what happens is that the window manager does not inform the applications - that it is shutting down and shuts down the display and conky screams. :-)

**Comment by su8** [comment]
Can be closed by #311

**Comment by lasers** [comment]
Closed via #588.
